modules/custom-status/custom-status.php

```diff
@@ -459,747 +466,806 @@ function post_admin_header() {
+			}
 
-		// Update each custom status with its new position
-		foreach ( $_POST['status_positions'] as $position => $term_id ) {
+			// Update each custom status with its new position
+			foreach ( $_POST['status_positions'] as $position => $term_id ) {
 
-			// Have to add 1 to the position because the index started with zero
-			$args = array(
-				'position' => (int)$position + 1,
-			);
-			$return = $this->update_custom_status( (int)$term_id, $args );
-			// @todo check that this was a valid return
+				// Have to add 1 to the position because the index started with zero
+				$args = array(
+					'position' => (int) $position + 1,
+				);
+				$return = $this->update_custom_status( (int) $term_id, $args );
+				// @todo check that this was a valid return
+			}
+			$this->print_ajax_response( 'success', $this->module->messages['status-position-updated'] );
 		}
-		$this->print_ajax_response( 'success', $this->module->messages['status-position-updated'] );
-	}
 
-	/**
-	 * Handle an Inline Edit POST request to update status values
-	 *
-	 * @since 0.7
-	 */
-	function ajax_inline_save_status() {
-		global $edit_flow;
+		/**
+		 * Handle an Inline Edit POST request to update status values
+		 *
+		 * @since 0.7
+		 */
+		public function ajax_inline_save_status() {
+			global $edit_flow;
 
-		if ( !wp_verify_nonce( $_POST['inline_edit'], 'custom-status-inline-edit-nonce' ) )
-			die( $this->module->messages['nonce-failed'] );
+			if ( ! isset( $_POST['inline_edit'] ) || ! wp_verify_nonce( $_POST['inline_edit'], 'custom-status-inline-edit-nonce' ) ) {
+				die( esc_html( $this->module->messages['nonce-failed'] ) );
+			}
 
-		if ( !current_user_can( 'manage_options') )
-			die( $this->module->messages['invalid-permissions'] );
+			if ( ! current_user_can( 'manage_options' ) ) {
+				die( esc_html( $this->module->messages['invalid-permissions'] ) );
+			}
 
-		$term_id = (int) $_POST['status_id'];
-		$status_name = sanitize_text_field( trim( $_POST['name'] ) );
-		$status_slug = sanitize_title( $_POST['name'] );
-		$status_description = stripslashes( wp_filter_nohtml_kses( trim( $_POST['description'] ) ) );
+			$term_id = isset( $_POST['status_id'] ) ? (int) $_POST['status_id'] : 0;
+			$status_name = isset( $_POST['name'] ) ? sanitize_text_field( trim( $_POST['name'] ) ) : '';
+			$status_slug = isset( $_POST['name'] ) ? sanitize_title( trim( $_POST['name'] ) ) : '';
+			$status_description = isset( $_POST['description'] ) ? stripslashes( wp_filter_nohtml_kses( trim( $_POST['description'] ) ) ) : '';
 
-		// Check if name field was filled in
-		if ( empty( $status_name ) ) {
-			$change_error = new WP_Error( 'invalid', __( 'Please enter a name for the status.', 'edit-flow' ) );
-			die( $change_error->get_error_message() );
-		}
+			// Check if name field was filled in
+			if ( empty( $status_name ) ) {
+				$change_error = new WP_Error( 'invalid', esc_html__( 'Please enter a name for the status.', 'edit-flow' ) );
```

Comment:
Nit: Escaping HTML in an error isn't usually necessary, especially as we escape it in the output line below. I assume this came from a linting error though, so maybe it made you do that.

Reply:
(same for the 5 `WP_Error`/`die()`s below)

Reply:
It's def the linter forcing me to do it, and I have the same qualm with it - why do it if it's being escaped already at the bottom? I wanted to reduce the amount of ignores that I had in place too tbh, so I just went ahead and did it